### PR TITLE
KAA-243: Customize Akka configuration for operations server

### PR DESCRIPTION
**Pull request by ashvayka**

### CODE REVIEW

**Comment by shpigor** [comment]
Reviewed

File: server/operations/src/main/java/org/kaaproject/kaa/server/operations/service/akka/AkkaContext.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    public AkkaContext() {
+        config = ConfigFactory.parseResources(AKKA_CONF_FILE_NAME).withFallback(ConfigFactory.load());
+    }
+    
+    public Config getConfig(){
+        return config;
+    }
+
+    public int getIOWorkerCount(){
+        return config.getInt("io_worker_count");
```
Comment (ghost) [inline server/operations/src/main/java/org/kaaproject/kaa/server/operations/service/akka/AkkaContext.java]: Please, extract to the named constant to get the same way as for AKKA_CONF_FILE_NAME.

File: server/operations/src/main/java/org/kaaproject/kaa/server/operations/service/akka/AkkaContext.java [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    }
+
+    public int getInactivityTimeout() {
+        return config.getInt("endpoint_actor_timeout");
```
Comment (ghost) [inline server/operations/src/main/java/org/kaaproject/kaa/server/operations/service/akka/AkkaContext.java]: Please, extract to the named constant to get the same way as for AKKA_CONF_FILE_NAME.

File: server/operations/src/main/java/org/kaaproject/kaa/server/operations/service/akka/actors/core/ApplicationActor.java [hunk 3]
```diff
@@ -492,36 +454,42 @@ private void processTermination(Terminated message) {
...
     }
 
-    private ActorRef getOrCreateUserVerifierActor(String name, EndpointUserService endpointUserService,
-            ApplicationService applicationService) {
+    private ActorRef getOrCreateUserVerifierActor() {
+        return getOrCreateUserVerifierActor(null);
+    }
+    
+    private ActorRef getOrCreateUserVerifierActor(String name) {
         ActorRef logActor = userVerifierSessions.get(name);
```
Comment (ghost) [inline server/operations/src/main/java/org/kaaproject/kaa/server/operations/service/akka/actors/core/ApplicationActor.java]: Please, rename logActor to something appropriate.

**Pull request merged**